k8s/namespaces/money-claims/cmc-claim-store/perftest.yaml

```diff
@@ -67,10 +61,7 @@ spec:
         MILO_CSV_SENDER: noreply@mail-money-claims-nonprod.platform.hmcts.net
         RPA_NOTIFICATIONS_BREATHINGSPACERECIPIENT: civilmoneyclaimsdemo@gmail.com
```

Comment:
Definitely using demo email?

Reply:
It is currently set to demo.
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/perftest/common/money-claims/claim-store.yaml